### DISCUSSION TITLE
BS 5 modals and interaction to next paint (INP) issue

### DISCUSSION THREAD

**Discussion by K-Thompson** [twbs/bootstrap#39784] [Q&A] [answered]
My site now fails Core Web Vitals for mobile. I've done some testing, and one action that is too slow is when the user opens a v5 modal. Has anyone else noticed this, and do you have a suggested fix? (This is only impacting my mobile, and just barely. Time to next interaction is about 235ms.)

**Comment by julien-deramond** [comment]
This is interesting. IDK why it would be different on mobile, except maybe if the expected values are different. On Bootstrap side, I'd say there shouldn't be any differences.
Do you know if it's due to the animation?
If so, maybe you could reduce the modal transition thanks to `$modal-transition` Sass variable which is set to `transform .3s ease-out;` by default.

**Reply by K-Thompson** [reply]
That's what I was thinking. Thank you for the insight about $modal-transition. I'll try to update BS and lower this value during the week and see it it makes a difference. I'll report back with my findings.

**Reply by K-Thompson** [reply, marked as answer]
The modals in question were displaying dynamic content (Highchart charts). Optimizing my code seems to have resolved the INP issue.
